source/linux/editing_files/vi.rst

```diff
@@ -0,0 +1,62 @@
+Vi
```

Comment:
Maybe you can reference the tutorial vim includes.

Reply:
I'm sorry, but I don't know exactly what you mean.

Reply:
There is a vim tutorial program, which ships with the most linux distros. 
Just type `vimtutor` in your terminal.